modules/nf-core/universc/README.md

```diff
@@ -45,10 +45,10 @@ process {
 
 ...
     withName: CELLRANGER_MKGTF {
-        container = "nfcore/universc:1.2.5.1"
+        container = "docker.io/biocontainers/universc:1.2.5.1"
     }
     withName: CELLRANGER_MKREF{
-       container = "nfcore/universc:1.2.5.1"
+       container = "docker.io/biocontainers/universc:1.2.5.1"
```

Comment:
is it correct to remove the `nfcore` here? It is maintained below at line 69

Reply:
Yeah this is hosted on nfcore!